src/web/src/license_POST.ecpp

```diff
@@ -125,24 +108,12 @@ UserInfo user;
             http_die ("internal-error", "Starting of start-db-services have failed. Consult system logs");
 
         // once done, check environment files for accessing the database
-        if (!shared::is_file (PASSWD_FILE))
+        if (!dbreadcredentails())
             http_die ("internal-error", "Database password file is missing");
-
-        // and setup db username/password
-        std::ifstream dbpasswd {PASSWD_FILE};
-        static char db_user[256];
-        memset (db_user, '\0', 256);
-        dbpasswd.getline (db_user, 256);
-        s_dropdq (db_user);
-        static char db_passwd[256];
-        memset (db_passwd, '\0', 256);
-        dbpasswd.getline (db_passwd, 256);
-        s_dropdq (db_passwd);
-        dbpasswd.close ();
-
-        putenv (db_user);
-        putenv (db_passwd);
-        dbpath ();
+    }else{
+        //enforce reload credential
+        if (!dbreadcredentails())
```

Comment:
If I get this changeset right, then beside relocating some code, this is the one effective logical change so far?

So now we create the license file during license_POST, and if a DB credential file already exists - re-read it (just in case it was not there or up-to-date when tntnet initially started), right?